app/Repositories/Eloquents/MessageHistoryRepository.php

```diff
@@ -24,4 +24,17 @@ public function getAllAndSearch($id, $keyword)
             return $query->paginate($perPage);
         }
     }
+
+    public function AdminGetAllAndSearch($id, $keyword)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR1.Methods.CamelCapsMethodName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/MessageHistoryRepository.php</br>**Line**: 28</br>Method name "MessageHistoryRepository::AdminGetAllAndSearch" is not in camel caps format